go.mod

```diff
@@ -5,6 +5,7 @@ go 1.21
 require (
 	github.com/crowdstrike/gofalcon v0.4.2
 	github.com/xeipuuv/gojsonschema v1.2.0
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
this was missed in prev version evidently. Not a deal breaker for the module, but got missed